src/delta.rs

```diff
@@ -382,6 +409,28 @@ fn prepare(line: &str, append_newline: bool, config: &Config) -> String {
     }
 }
 
+fn _line_number_style(line_number: Option<usize>) -> String {
+    match line_number {
+        Some(x) => format!("{:^4}", x),
+        None => format!("    "),
+    }
+}
+
+pub fn prepare_line_numbers(
+    line_number_in: Option<usize>,
+    line_number_out: Option<usize>,
+    config: &Config,
+) -> String {
+    let formatted = format!(
+        "{}⋮{}",
+        _line_number_style(line_number_in),
+        _line_number_style(line_number_out),
+    );
+    let numbers = paint::paint_text_foreground(&formatted, config.hunk_color, config.true_color);
+    let divider = paint::paint_text_foreground("│", config.hunk_color, config.true_color);
+    format!("{}{}", numbers, divider)
+}
+
```

Comment:
Playing around with the vertical dividing line character. E.g. https://jrgraphix.net/r/Unicode/2500-257F https://unicode.org/charts/nameslist/n_2500.html. Any thoughts? (I haven't found one yet which yields a vertical dashed line without "breaks" between lines.)

**Current:**
<table><tr><td><img width=300px src="https://user-images.githubusercontent.com/52205/81964464-9e59f880-95e4-11ea-9fae-7c82c294b8fe.png" alt="image" /></td></tr></table>

**Alternatives:**

<table><tr><td><img width=300px src="https://user-images.githubusercontent.com/52205/81964335-69e63c80-95e4-11ea-9c7d-7c94fb133d18.png" alt="image" /></td></tr></table>

`U2506`

<table><tr><td><img width=300px src="https://user-images.githubusercontent.com/52205/81964050-11af3a80-95e4-11ea-9d28-990a7caf93f4.png" alt="image" /></td></tr></table>

`U250A`

<table><tr><td><img width=300px src="https://user-images.githubusercontent.com/52205/81964940-57b8ce00-95e5-11ea-8607-86a8ff1b15db.png" alt="image" /></td></tr></table>

Reply:
Added an option to input a divider (`--number-center-divider-string`).